docs/template/edx/publish.js

```diff
@@ -629,7 +629,7 @@ exports.publish = (memberData, opts, tutorials) => {
         generateSourceFiles(sourceFiles, opts.encoding);
     }
 
-    // if (members.globals.length) { generate('Global', [{kind: 'globalobj'}], globalUrl); }
+    if (members.globals.length) { generate('Global', [{kind: 'globalobj'}], globalUrl); }
```

Comment:
[inform] generates the `global.html` file which is used in links for custom `@typedef` types used within the React module. without this, the generated JSDoc link results in a 404.